docs/learn/learn-balance-transfers.md

```diff
@@ -38,34 +39,66 @@ for it to remain active.** See the [existential deposit][] section for more info
 After setting your inputs correctly, click the "Make Transfer" button and confirm. Once the transfer
 is included in a block you will see a green notification in the top-right corner of your screen.
 
+### Signing on Polkadot Extension
+
+If your account is injected to Polkadot JS UI through the Polkadot Extension, the balance transfer transaction needs to be signed
+on the popup window. You can examine the method and info dropdowns highlighted in the screenshot below and ensure they match with the original transaction submitted. Sign the transaction by entering the password for the account set for this account when it was created on the Polkadot Extension.
+
+![extension transfer](../assets/accounts/extenstion-transaction.png)
+
+### Signing on Ledger device
+
+Assuming that you imported your Ledger account to Polkadot Extension, you will be presented with a popup window for signing the balance transfer transaction. Review the transaction details on the Ledger device and approve it to sign the transaction.
+
+> If you are looking for instructions in a video, watch the tutorial on how to connect [Ledger to Polkadot JS UI](https://youtu.be/7VlTncHCGPc).
+
+![Ledger transfer](../assets/accounts/ledger-transaction.png)
+
+### Signing on Parity Signer App
+
+If you imported your Parity Signer account to Polkadot JS UI directly, you will be presented with a popup window with a QR code and a camera window as shown below.
+Open QR scanner on your phone's Parity Signer app and scan the QR code presented. You may be prompted to enter your PIN to sign the transaction.
+The signed extrinsic will then be available as a QR code that needs to be shown in front of your computer's camera (in the red square window shown to the right in the screenshot below).
+
+![Companion transfer](../assets/accounts/signer-transaction.png)
+
+If you imported your Parity Signer account to [Parity Signer Companion](https://parity.link/signer-companion), you will be presented with a popup window with a QR code as shown below.
```

Comment:
```suggestion
If you import your Parity Signer account to [Parity Signer Companion](https://parity.link/signer-companion), you will see a pop-up window with a QR code, as shown below.
```

Reply:
This changes the meaning of the original sentence. There are two different ways the account could have been imported - 1. Parity Signer Companion 2. Directly to the Polkadot-JS UI. Probably I'll have to re-phrase it.
```suggestion
If your account was imported to [Parity Signer Companion](https://parity.link/signer-companion), you will see a pop-up window with a QR code, as shown below.
```